airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DatabaseSchema.java

```diff
@@ -50,6 +50,9 @@
   // AirbyteMetadata
   AIRBYTE_METADATA("AirbyteMetadata.yaml");
 
+  // TODO: add airbyte_configs table
+  // AIRBYTE_CONFIGS("AirbyteConfigs.yaml");
```

Comment:
TODO reminder bot

Reply:
I don't plan to address this TODO in this PR. Adding this enum will break the auto migration tests. I will look into it in the migration project.